<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcptool</class>
 <widget class="QDialog" name="tcptool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lb_tcptitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TCP Debug Tool</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le_ip"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_tcpconnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="te_sendmsg"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pb_send">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="te_recvmsg"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
